test/popcorn/eval_test.exs

```diff
@@ -607,8 +607,9 @@ defmodule Popcorn.EvalTest do
     |> AtomVM.assert_result([2, 3, 4, 5])
   end
 
+  @tag :skip
```

Comment:
We need to skip this because https://github.com/atomvm/AtomVM/pull/1804 isn't included in https://github.com/software-mansion-labs/FissionVM/pull/151. I'm not sure whether to include it - but if so, maybe in a separate PR